common/djangoapps/student/tests/tests.py

```diff
@@ -2,19 +2,21 @@
 """
 Miscellaneous tests for the student app.
 """
-from datetime import datetime, timedelta
 import logging
-import pytz
 import unittest
 import ddt
+from datetime import datetime, timedelta
+
+import pytz
+from mock import Mock, patch
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from urlparse import urljoin
```

Comment:
Minor: `urlparse` is part of the standard library and belongs with the import above.